src/style/table.rs

```diff
@@ -68,3 +66,30 @@ pub enum TableComponent {
     BottomLeftCorner,
     BottomRightCorner,
 }
+
+impl TableComponent {
+    pub fn components() -> impl Iterator<Item = TableComponent> {
```

Comment:
Please change this to `iter`, otherwise this would effectively be a breaking change as it changes the API surface of `TableComponent`.

Reply:
Updated.